tron/core/actionrun.py

```diff
@@ -681,6 +684,8 @@ def _kill_mesos_task(self):
             mesos_cluster = MesosClusterRepository.get_cluster(
                 self.mesos_address
             )
+            if self.task_id is None:
+                raise AttributeError
```

Comment:
Hehe. Please do a bit more. 
Actually, are you sure it should error hard like this? Maybe there won't be a task id sometimes. maybe it should just return an error string that there was no task id to kill?

Reply:
Yeah, you're right. Return an error string makes more sense. I'll change that.